emf/src/test/resources/properties-language.json

```diff
@@ -9,6 +9,11 @@
   "nodes": [
     {
       "id": "io_lionweb_Properties",
+      "classifier": {
```

Comment:
As for the other file, I would consider leaving the file as-is, to demonstrate that the test should pass anyway and it is not dependent on the order of keys

Reply:
Shouldn't we make a separate test case (and file) for that, then?

Reply:
Yes, I think so. I will mark this as resolved then